AIDriver.lua

```diff
@@ -423,7 +423,7 @@ function AIDriver:driveCourse(dt)
 		self:setSpeed(self:getRecordedSpeed())
 	end
 	local isInTrigger, isAugerWagonTrigger = self.triggerHandler:isInTrigger()
-	if self:getIsInFilltrigger() or isInTrigger then
```

Comment:
isInTrigger is used for triggerSpeeds near triggers in mode 1/4/8 ...
we still need this one

Reply:
But the local is never assigned a value. May I ask what are you checking there?

Reply:
function TriggerHandler:isInTrigger()
	local bool = next(self.triggers) ~=nil
	return bool, self.isInAugerWagonTrigger
end
return 2 values

Reply:
@GeorgeEarslight I think @schwiti6190 is right.

Reply:
Ah, I see, thanks for the explanation.
I'm not used to languages with multiple return variables.
Sorry for dragging this around.